packages/core/src/actors/index.ts

```diff
@@ -27,7 +27,8 @@ const emptyLogic = fromTransition((_) => undefined, undefined);
 
 export function createEmptyActor(): ActorRef<
   Snapshot<undefined>,
-  AnyEventObject
+  AnyEventObject,
+  any
```

Comment:
Would it still work with this?
```suggestion
  AnyEventObject
```